modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/ScheduleContainer.tsx

```diff
@@ -68,6 +77,48 @@ export default function ScheduleContainer({
 		>
 			<ClayPanel.Body className="lfr-object__entries-schedule-panel">
 				<div className="row">
+					<ScheduleField
```

Comment:
Hi @rtienpal, could you check if we can iterate to render the `ScheduleField`? Check the [react documentation](https://legacy.reactjs.org/docs/lists-and-keys.html#rendering-multiple-components) about that/